src/animation/SetStyle.ts

```diff
@@ -0,0 +1,67 @@
+import { Animation } from "./Animation.js";
+import * as THREE from "three";
+import Shape, { Style } from "../geometry/Shape.js";
+
+export default class SetStyle extends Animation {
+  private initialStyle: Style = {};
+  private targetStyle: Style = {};
```

Comment:
There's no need to set an initial value for either of these. Also, include a currentStyle here rather than inside the animation function so that it isn't recreated on each frame.